src/Sentry.NLog/Sentry.NLog.csproj

```
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NLog" Version="4.5.11" />
+    <PackageReference Include="NLog" Version="5.2.4" />
```

I wouldn't go to the latest version. How about the latest patch on the minor `0`:

https://www.nuget.org/packages/NLog/5.0.5?
```suggestion
    <PackageReference Include="NLog" Version="5.0.5" />
```